metadata-ingestion/src/datahub/ingestion/source/snowflake/snowflake_usage_v2.py

```diff
@@ -451,17 +451,10 @@ def _get_operation_aspect_work_unit(
                 yield wu
 
     def _process_snowflake_history_row(
-        self, row: Any
+        self, event_dict: dict
```

Comment:
We use DictCursor in snowflake, so this is always a dictionary.

Reply:
nice